<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rattrap</class>
 <widget class="QMainWindow" name="Rattrap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_mode">
             <property name="text">
              <string>Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radiobtn_mode1">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>F3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radiobtn_mode2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>F4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radiobtn_mode3">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>F5</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_color">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="color">
             <item>
              <property name="text">
               <string>Black</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Blue</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cyan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Red</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Magenta</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Yellow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>White</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Green</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_rate">
             <property name="text">
              <string>Report Rate:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rate">
             <item>
              <property name="text">
               <string>125</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>250</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>500</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1000</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>DPI Shift:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dpi_shift">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_dpi1">
             <property name="text">
              <string>DPI #1:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dpi1">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_dpi2">
             <property name="text">
              <string>DPI #2:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dpi2">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_dpi3">
             <property name="text">
              <string>DPI #3:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dpi3">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_dpi4">
             <property name="text">
              <string>DPI #4:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dpi4">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_left">
             <property name="text">
              <string>Left</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="left">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_right">
             <property name="text">
              <string>Right</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="right">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_middle">
             <property name="text">
              <string>Middle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="middle">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_g4">
             <property name="text">
              <string>G4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="g4">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_g5">
             <property name="text">
              <string>G5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="g5">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_g6">
             <property name="text">
              <string>G6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="g6">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_g7">
             <property name="text">
              <string>G7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="g7">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_g8">
             <property name="text">
              <string>G8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="g8">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_g9">
             <property name="text">
              <string>G9</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="g9">
             <property name="text">
              <string>Not Defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="button_apply">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Apply Changes!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>356</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
